web/src/i18n/locales/en.json

```
@@ -1515,6 +1515,14 @@
     "数据看板更新间隔": "Data dashboard update interval",
     "数据看板设置": "Data dashboard settings",
     "数据看板默认时间粒度": "Data dashboard default time granularity",
+    "默认显示模式": "Default Display Mode",
+    "显示金额": "Show Amount",
+    "显示 Token": "Show Tokens",
+    "金额": "Amount",
```

Biome (2.4.6)

[error] The key 金额 was already declared.

(lint/suspicious/noDuplicateObjectKeys)